pkgs/applications/misc/prusa-slicer/default.nix

```diff
@@ -192,5 +192,9 @@ stdenv.mkDerivation (finalAttrs: {
     maintainers = with maintainers; [ moredread tweber tmarkus ];
   } // lib.optionalAttrs (stdenv.isDarwin) {
     mainProgram = "PrusaSlicer";
+    # Reported on 2024-01-21
+    # Error: binary_function can't be found
```

Comment:
If there is no upstream fix for the deprecated `std::binary_function`, downgrading Clang might help. This is typically done with

```nix
  prusa-slicer = darwin.apple_sdk_11_0.callPackage ../applications/misc/prusa-slicer {
    stdenv = if stdenv.isDarwin then overrideLibcxx llvmPackages_14.stdenv else stdenv;
  };
```

Didn't test whether this improves the situation.